python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -374,6 +375,9 @@ def _get_places(self):
                 return []
         places = [fluid.CPUPlace()]
         cpu_only = self._cpu_only if hasattr(self, '_cpu_only') else False
+        use_ngraph = bool(os.getenv("FLAGS_use_ngraph", False))
+        if use_ngraph:
+            cpu_only = True
```

Comment:
If `cpu_only=True` here, do you remove all the `self._cpu_only = True` in `test_xxx_ngraph_op.py`? Such as https://github.com/PaddlePaddle/Paddle/blob/4da291c6a38f532302b4996b1a078dfa11f9bc7e/python/paddle/fluid/tests/unittests/ngraph/test_mean_ngraph_op.py#L23